addons/udes_stock/models/stock_picking.py

```diff
@@ -37,6 +37,14 @@ class StockPicking(models.Model):
 
     _order = "priority desc, scheduled_date asc, sequence asc, id asc"
 
+    # Adding fields to be hidden by default from form view.
+    DetailedFormViewFields = [
+        "u_first_picking_ids",
+        "u_prev_picking_ids",
+        "u_next_picking_ids",
+        "u_created_back_orders",
+    ]
```

Comment:
Sorry, something I missed before... can we add the `_qty` fields here? Otherwise they will be computed even in simple view.